### PR TITLE
Azure Theme Constants Clean Up

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
## New Behavior
No visual changes. Alphabetized the Azure Theme constants and renamed choiceGroupHeight to choiceFieldHeight to be applicable to multiple selection controls (choicegroup, checkbox). 
image

## Related Issue(s)
Allows partner teams like Elixir to theme their controls when Azure Theme is applied.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/7af0c8852e037dac9ff29e9b1eb4641cc4acaeea'>7af0c8852e037dac9ff29e9b1eb4641cc4acaeea</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26013&commitId=dcc81a594360a40133fc6c968aea3f7d704d2e48&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by robtaft-ms — APPROVED** [review]
This looks good, thank you!
In the future a full rework/refactor of the naming with some Dev input would be great, this is good for now.

File: packages/azure-themes/src/azure/Constants.ts [hunk 1]
```diff
@@ -5,25 +5,22 @@ import {
...
-export const fontWeightBold = '700';
 export const inputControlHeight = '24px';
-export const choiceGroupDimensions = '18px';
-export const inputControlPadding = '12px';
 export const inputControlHeightInner = '20px';
-export const textAlignCenter = 'center';
-export const transparent = 'transparent';
+export const inputControlPadding = '12px';
 export const MinimumScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
 export const MediumScreenSelector = getScreenSelector(ScreenWidthMinMedium, ScreenWidthMaxMedium);
```
Comment (khmakoto) [inline packages/azure-themes/src/azure/Constants.ts:24]: These 2 should be swapped if you want to maintain alphabetical order.

```suggestion
export const MediumScreenSelector = getScreenSelector(ScreenWidthMinMedium, ScreenWidthMaxMedium);
export const MinimumScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**